orchestrator/orchestratorAutomation.ps1

```diff
@@ -16,7 +16,11 @@
     ./orchestratorAutomation.ps1 -aifip ww.xx.yy.zz -orcname orchestrator.uipath.com
 
     If ai-app is accessed via domain instead of IP:PORT combo, then enable domainBasedAccess to true
-    .\orchestratorAutomation.ps1 -aifip "aif-sahil-aks.westeurope.cloudapp.azure.com" -orcname "aifabricdevorch.northeurope.cloudapp.azure.com" -domainBasedAccess "true"
+    .\orchestratorAutomation.ps1 -aifip "aif-sahil-aks.westeurope.cloudapp.azure.com" -orcname "aifabricdevorch.northeurope.cloudapp.azure.com" -portlessAccess "true"

```

Comment:
shouldn't we have if else here? Are we expecting users to comment one of these lines before running?

Reply:
btw, are we taking a stance that we will only support orch 20.10? If not, then let's have existing script too.

Reply:
@vivekjhauipath we are supporting both 20.4 and 20.10 config edit... the script is backward compatible with 20.4 as well ... and regarding your first comment , it's just documentation on various ways on how can we run that script